custom/icds_reports/views.py

```diff
@@ -842,3 +843,34 @@ def get(self, request, *args, **kwargs):
         return JsonResponse(data={
             'report_data': data,
         })
+
+
+@method_decorator([login_and_domain_required], name='dispatch')
+class EnrolledChildrenView(View):
+    def get(self, request, *args, **kwargs):
+        step = kwargs.get('step')
+        now = datetime.utcnow()
+        month = int(self.request.GET.get('month', now.month))
+        year = int(self.request.GET.get('year', now.year))
+        day = int(self.request.GET.get('day', now.day))
+        test_date = datetime(year, month, day)
```

Comment:
I think this is in a few places but this will break if you try to look for data in june but today is july 31 because june does not have 31 days